python/tvm/relay/op/strategy/generic.py

```diff
@@ -1395,3 +1395,28 @@ def cumsum_strategy(attrs, inputs, out_type, target):
         name="cumsum.generic",
     )
     return strategy
+
+
+def wrap_compute_unique(topi_compute):
+    """Wrap unique topi compute"""
+
+    def _compute_unique(attrs, inputs, _):
+        return topi_compute(inputs[0])
```

Comment:
Any reason the shape func is defined but not being used here?

Reply:
Since `output[0]`,`output[1]`,`output[2]` have the same shape as `input[0]` and `output[3]` has shape `[1]`. We don't need the shape function in compute function.